flink-staging/flink-hbase/src/test/resources/hbase-site.xml

```diff
@@ -22,14 +22,13 @@
 -->
 <configuration>
 
+    <!-- 
```

Comment:
I think you can remove the hbase-site.xml file. It is required only if you have hbase settings different from the default ones. Also the log4j.properties could be removed